packages/suite/src/actions/wallet/coinmarketBuyActions.ts

```diff
@@ -85,6 +89,33 @@ export const saveBuyInfo = (buyInfo: BuyInfo): CoinmarketBuyAction => ({
     buyInfo,
 });
 
+export const convertDrafts = () => (dispatch: Dispatch, getState: GetState) => {
+    const { router, wallet } = getState();
+    const { selectedAccount, settings } = wallet;
+
+    if (router.route?.name === 'wallet-coinmarket-buy') {
+        return;
+    }
+    if (!selectedAccount || selectedAccount.status !== 'loaded') {
+        return;
+    }
+
+    const getDraft = formDraftActions.getDraft<SellFormState>('coinmarket-buy');
```

Comment:
Could `convertDrafts` be common to buy and sell (and exchange)? They seem almost identical. It would also fix the copy/paste bug of using `<SellFormState>` in the buy version.

Reply:
We checked with Luky and we could have a common method for buy and sell but not for exchange. Since the deadline is pretty close, I'd like to get a working solution ASAP and focus on refactoring later.